src/pages/signup/Form4.tsx

```diff
@@ -0,0 +1,84 @@
+import { useNavigate } from "react-router-dom";
+import React, { useState, useRef } from "react";
+
+
+function Form4() {
+    const [otp, setOTP] = useState<string[]>(Array(5).fill(""));
+  const inputRefs = useRef<(HTMLInputElement | null)[]>([]);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>, index: number) => {
+    const value = e.target.value;
+
+    if (!isNaN(Number(value))) {
+      setOTP((prevOTP) => {
+        const newOTP = [...prevOTP];
+        newOTP[index] = value;
+        return newOTP;
+      });
+
+      // Move focus to the next input field
+      if (value !== "" && index < otp.length - 1) {
+        inputRefs.current[index + 1]?.focus();
+      }
+    }
+  };
+
+  const handlePaste = (e: React.ClipboardEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const pasteData = e.clipboardData.getData("text/plain");
+
+    if (/^\d+$/.test(pasteData)) {
+      setOTP((prevOTP) => {
+        const newOTP = [...prevOTP];
+        for (let i = 0; i < newOTP.length; i++) {
+          if (pasteData[i]) {
+            newOTP[i] = pasteData[i];
+          }
+        }
+        return newOTP;
+      });
+    }
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>, index: number) => {
+    if (e.key === "Backspace" && otp[index] === "" && index > 0) {
+      inputRefs.current[index - 1]?.focus();
+    }
+  };
+
+    const router = useNavigate();
+
+    const handleVerification = () => {
```

Comment:
This function is unused. Kindly, comment it out or prefix it with an underscore.

Reply:
ok ok just saw this